package.json

```diff
@@ -5,10 +5,10 @@
   "scripts": {
     "lint": "cd ./react && yarn lint",
     "format": "prettier --write \"**/*.{ts,js,json}\"",
-    "test": "cd ./react && yarn test",
+    "test": "cd ./react && yarn test --verbose",
```

Comment:
Any reason for the `--verbose` flag?

Reply:
Just personal preference 😅 
I really like the `--verbose` output, since it shows each individual test case